problems/my_rotate.rb

```diff
@@ -1,5 +1,9 @@
 class Array
-  def my_rotate(positions)
+  # Define a method `my_rotate(positions)` that rotates the 
```

Comment:
Should be `Array#my_rotate` or otherwise specify that it's monkey patched onto the Array class